app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  after_save :update_comments_counter
+
+  def update_comments_counter
+    post.increment!(:comments_counter)
+  end
```

Comment:
- [Required] Good job using `Rails Active Callbacks`, I think it would be much better if you made the `update_comments_counter` method private since it should only be called within this class. You can do this by using the code below.

```Ruby
class Comment < ApplicationRecord
  belongs_to :author, class_name: 'User'
  belongs_to :post

  after_save :update_comments_counter
  
  private
  def update_comments_counter
    post.increment!(:comments_counter)
  end
end

```